media-nep/src/main/java/nl/vpro/nep/service/impl/NEPGatekeeperServiceImpl.java

```diff
@@ -249,9 +260,9 @@ public WorkflowExecution transcode(
                     throw new IllegalStateException(closeableHttpResponse.getStatusLine() + ":" + w.toString());
 
             }
+        } catch (IOException e) {
+            throw new NEPException(e, "Exception from a NEP function call");
```

Comment:
De message lijkt altijd  te zijn 'Exception from a NEP function call'. Dat lijkt me dan onzin. Laten we er iets informatiefs neezetten, of anders kunnen we nog beter e.getMessage() doen misschien?